### ISSUE TITLE
Clarification needed in the definition of a scan

### ISSUE THREAD

**Issue by pkeller** [pdbxmmcifwg/diffrn-data-set-extension#4]
https://github.com/pdbxmmcifwg/diffrn-data-set-extension/blob/614078f5fe732b071ec929a875661b697fda0769/diffrn-data-set-extension-v2.dic#L1986-L1991

Does "contiguous" in this context mean:
1. contiguously collected, or
1. contiguous in scan axis angle

This makes a big difference for interleaved data collections (inverse-beam, wavelength-interleaved MAD), where a contiguously-collected set of images is typically about 10°.

I suggest that the description should be amended to read "contiguously collected": even where such scans can be assembled into larger sets of images that are contiguous in the scan axis angle, the boundaries between the contiguously collected images are often visible in data processing (e.g. as discontinuities in scale factors).

**Comment by dagewa** [comment]
I agree - a scan should be contiguous in dose and angle. The DIALS concept is that scan-dependent parameters can be reasonably modelled with smooth curves, and discontinuities form the boundaries between separate scans.
